code/modules/mob/living/carbon/human/emote_fulp_edit.dm

```diff
@@ -0,0 +1,9 @@
+/datum/emote/living/carbon/human/scream
+	key = "scream"
+	key_third_person = "screams"
+	message = "screams!"
+	message_mime = "acts out a scream!"
+	emote_type = EMOTE_AUDIBLE

```

Comment:
dont need this if youre editing the emote i assume? these are on the base thing

Reply:
I just want to make sure this doesn't ditch a lot of the code from the original scream.

Reply:
but it will now. it will ditch any updates made to the original scream upstream